unit-tests/func/mur/l500/test-change-preset.cpp

```diff
@@ -0,0 +1,32 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
```

Comment:
2021 :)

Reply:
I hope I will not need to update to 2022 in this session :)

Reply:
Updated